.eslintrc

```diff
@@ -9,7 +9,8 @@ env:
 
 rules:
   no-console: 0  # This project purposely uses console.
-
+  space-before-function-paren: ["error","never"]
+  padded-blocks: ["error", "never"]
```

Comment:
No. "never" means the option is never allowed. Padded blocks are "never" allowed. 
http://eslint.org/docs/rules/padded-blocks 

This is also how Travis CI wants the style to be. It does not mean the rule is never enforced.